packages/core/graph/src/Graph.js

```diff
@@ -506,7 +500,7 @@ export default class Graph<TNode, TEdgeType: number = 1> {
    *
    * This replaces `dfs` and will replace `dfsFast`.
```

Comment:
What will be the approach for replacing `dfsFast`? Is it just changing the call sites? Will that require a separate flag / testing to ensure functionality is equivalent and performance doesn't regress?

Reply:
We will need another flag & a bit more code